grpc-client-spring-boot-starter/src/main/java/net/devh/boot/grpc/client/config/GrpcChannelProperties.java

```diff
@@ -53,6 +53,33 @@
 @EqualsAndHashCode
 public class GrpcChannelProperties {
 
+    // --------------------------------------------------
+    // Target Deadline
+    // --------------------------------------------------
+
+    private Duration deadline = null;
+
+    /**
+     * Gets the connection deadline.
+     *
+     * @return The connection deadline or null
+     * @see #setDeadline(Duration)
+     */
+    public Duration getDeadline() {
+        return this.deadline;
+    }
+
+    /**
+     * Set the deadline for the stub. If nothing is configured then the deadline will not be used by default
```

Comment:
```suggestion
     * Set the deadline for the stub. If nothing is configured then the deadline will not be used by default.
```

Maybe mention what happens if zero is set.